_projects/03-efficient-starai.md

```diff
@@ -39,11 +39,17 @@ excerpt: "Efficient Starai excerpt"
 ## 3. Lifted Continuous Inference
 
 ### People Involved:
+* Yuqiao Chen
 
 ### About:
+Continuous values are common in the real world, but in traditional graphical model, inference with continuous variable may requires to compute integal of potential functions without tractable close form. Because of this, lots of works about inference in continuous domain limits the potential to some special exponential family, which cannot fully represent the complexity of real world. Some other works focus on using sampling based method for inference. These method are powerful in the domain with both continuous and discrete values (Hybrid Model), but they are too computation-expensive to be applied to real word application when large number of variables are involved.
 
-### Sample Publications
+In our work, we are trying to combine continuous inference with lifted inference to improve the efficiency so that it can be applied into real word task such as early detection of Alzehmer's Disease through doing inference on brain fMRI features.
+
+*[Collaborators: **Prof. Nicholas Ruozzi** - University of Texas at Dallas]*
```

Comment:
This would work, but maybe Prof. Ruozzi could be added instead to the *People Involved* section?